server/init.sql

```diff
@@ -1,3 +1,12 @@
 CREATE DATABASE IF NOT EXISTS `veryrezsi`;
 CREATE USER IF NOT EXISTS 'veryrezsi'@'%' IDENTIFIED BY 'password';
-GRANT ALL PRIVILEGES ON veryrezsi.* TO 'veryrezsi'@'%';
\ No newline at end of file
+GRANT ALL PRIVILEGES ON veryrezsi.* TO 'veryrezsi'@'%';
+
+# Account activation remover scheduled event
+SET GLOBAL event_scheduler = ON;
+CREATE EVENT veryrezsi.account_activation_remover
+    ON SCHEDULE
+		EVERY 1 DAY
+    COMMENT 'Removes expired account_activation rows, once a day'
+    DO
+		DELETE FROM veryrezsi.account_activation WHERE expiration < now();
```

Comment:
Perfect solution.
remover sounds silly IMO. What about "garbage collector"?

Reply:
That somehow sounds worse, but I like it.